packages/core/README.md

```diff
@@ -0,0 +1,4 @@
+# @smithy/core
```

Comment:
Is this package intended to remain internal? Can we note what this package is for and that it likely should not be relied up directly?

Reply:
Good catch, I will add more details in the README.